.travis.yml

```diff
@@ -6,6 +6,10 @@ rvm:
   - 2.3
   - 2.4
 
+before_install:
```

Comment:
I've updated the travis configuration to get Rails 4.2.x green and I while I don't generally like using older versions, I think it's acceptable here in order to maintain testing 4.2 (at least until we determine we don't want it supported anymore).

Reply:
I've sent a different fix for this issue here that only downgrades bundler for Rails 4.2 (and not other versions of Rails). https://github.com/basecamp/marginalia/pull/90/files